docs/pages/docs/05-distributed-analytics/compute-statistics.md

```diff
@@ -29,4 +30,115 @@ For example, the following query executes a distributed computation to determine
 compute mean of age in person;
 ```
 
-The [Compute Queries](./compute-queries) documentation covers the usage of `compute` in more detail.
+The [Statistics Queries](./compute-statistics) documentation covers the usage of statistics in more detail.
+
+
+## Available Statistics Methods
+
+The following methods are available to perform simple statistics computations, 
+and we aim to add to these as demand dictates. Please get
+in touch on our [discussion](https://discuss.grakn.ai/) page to request any features that are of particular interest
+to you. A summary of the statistics algorithms is given in the table below.
+
+| Algorithm | Description                                   |
+| ----------- | --------------------------------------------- |
+| [`count`](#count)     | Count the number of instances.                        |
+| [`max`](#maximum)    | Compute the maximum value of an attribute. |
+| [`min`](#minimum)    | Compute the minimum value of an attribute. |
+| [`mean`](#mean)    | Compute the mean value of an attribute.                           |
+| [`median`](#median)    | Compute the median value of an attribute.                           |
+| [`std`](#standard-deviation)    | Compute the standard deviation of an attribute. |
+| [`sum`](#sum)    | Compute the sum of an attribute. |
```

Comment:
At the top of this page, we already have a list of methods. I'd remove that list and just keep this nice table!

Reply:
👍